README.md

```diff
@@ -93,6 +74,81 @@ IPFS has a name service:
+
+## Demos & Examples
+
+* [container-demos](https://github.com/ipfs/container-demos)
+    Demos on how to boot docker images and VMs.
+
+* [examples](https://github.com/ipfs/examples)
+    Examples on how to use go-ipfs.
+
+
+# Language Implementations
+
+For a list of IPFS API Client Libraries, please refer to *[Client Libraries](#client-libraries)*
+
+| Language | Project | Completeness |
+|----------|---------|--------------|
+| Go (reference) | https://github.com/ipfs/go-ipfs | |
+| Javascript | https://github.com/ipfs/js-ipfs | |
+| Python | https://github.com/ipfs/py-ipfs | |
+| Haskell | https://github.com/cleichner/haskell-ipfs | |
+| .net | https://github.com/wasabii/dotnet-ipfs | |
+| C++ | https://github.com/subtly/cpp-ipfs | |
+| Rust | https://github.com/rschulman/rust-ipfs | |
+
+Would you like to start your own language implementation of IPFS? Check out the [IPFS Implementation Guide](https://github.com/ipfs/specs/blob/overviews/overviews/implement-ipfs.md). There is at present no "perfect spec" and many protocol details are still to be decided upon.
+
+## Client Libraries
+
+For a list of IPFS implementations, please refer to *[Language Implementations](#language-implementations).
+
+| Language | Client Library | Completeness |
+|----------|----------------|--------------|
+| Go | https://github.com/ipfs/go-ipfs-api | |
+| Java | https://github.com/ipfs/java-ipfs-api | |
+| Javascript | https://github.com/ipfs/js-ipfs-api | |
+| Python | https://github.com/ipfs/python-ipfs-api | |
+| Scala | https://github.com/ipfs/scala-ipfs-api | |
+| Haskell | https://github.com/davidar/hs-ipfs-api | |
+| Swift | https://github.com/ipfs/swift-ipfs-api | |
+| CommonLisp | https://github.com/WeMeetAgain/cl-ipfs-api | |
+| Rust | https://github.com/rschulman/rust-ipfs-api | |
+| Ruby | https://github.com/Fryie/ipfs-ruby | |
+| Swift | https://github.com/NeoTeo/ipfs-osx-service | |
+| PHP | https://github.com/cloutier/php-ipfs-api | |
+| C/C++ | contact: [@PayasR](https://github.com/PayasR) | 0% |
+| Julia | contact: [@rened](https://github.com/rened) | 0% |
+| Lua | contact: [@seclorum](https://github.com/seclorum) | 0% |
+| Erlang | ! | 0% |
+| Objective C | ! | 0% |
+
+Want to contribute one of the above client libraries? [Tell us](https://github.com/ipfs/ipfs/issues/83) so we can help!
+
+# Community Projects
+
+| Project | Description |
+|---------|-------------|
+| [astralboot](https://github.com/ipfs/astralboot) | A low level boot server that deploys directly out of IPFS.|
+| [ipfs-web-app](https://github.com/ipfs/ipfs-web-app) | IPFS web app interface. |
+| [starlog](https://github.com/ipfs/starlog) | Logging the development of an interplanetary filesystem. This will evenutlaly be a way of making blog posts with IPFS. |
+| [station](https://github.com/ipfs/station) | Electron Shell based IPFS app. |
+| [webui](https://github.com/ipfs/webui) | The IPFS webui accessible from the api gateway typically at port 500.1 |
```

Comment:
"list" as in the table format? Should this be moved to ipfs/apps? Or should it be more like awesome-lists?